### PR TITLE
Macroified ACE_Zeus Strings

### PR DESCRIPTION
**Pull request by SilentSpike**
Pretty self explanatory, also changed the disabled string to use the existing A3 one

### CODE REVIEW

**Comment by jonpas** [comment]
:+1:

File: addons/zeus/stringtable.xml [hunk 1]
```diff
@@ -49,10 +49,6 @@
             <English>Reveal mines to allies and place map markers.</English>
             <Polish>Pokazuj znaczniki min dla sojuszników i twórz markery na mapie w miejscu min.</Polish>
         </Key>
-        <Key ID="STR_ACE_Zeus_revealMines_disable">
-            <English>Disabled</English>
-            <Polish>Wyłączone</Polish>
-        </Key>
```
Comment (Glowbal) [inline addons/zeus/stringtable.xml]: Is this supposed to be removed?

Comment (SilentSpike) [inline addons/zeus/stringtable.xml]: Yeah, I'm using the translation provided by A3

Comment (Glowbal) [inline addons/zeus/stringtable.xml]: :+1:

**Pull request merged**